tasks/extract_filepaths_for_granule/README.md

```diff
@@ -2,41 +2,163 @@
+
+
+## Input
+The lambda handler event excepts a dictionary having a list of granules as input.
+```
+{
+      "granules":[
+         {
+            "granuleId":"MOD09GQ.A0219114.N5aUCG.006.0656338553321",
+            "dataType":"MOD09GQ_test-jk2-IngestGranuleSuccess-1558420117156",
+            "version":"006",
+            "files":[
+               {
+                  "key":"MOD09GQ___006/2017/MOD/MOD09GQ.A0219114.N5aUCG.006.0656338553321.h5",
+                  "bucket":"cumulus-test-sandbox-protected",
+                  "fileName":"MOD09GQ.A0219114.N5aUCG.006.0656338553321.h5",
+                  "path":"jk2-IngestGranuleSuccess-1558420117156-test-data/files",
+                  "url_path":"{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{extractYear(cmrMetadata.Granule.Temporal.RangeDateTime.BeginningDateTime)}/{substring(file.name, 0, 3)}",
+                  "type":"data",
+                  "duplicate_found":"True",
+                  "size":1098034
+               },
+               {
+                  "key":"MOD09GQ___006/MOD/MOD09GQ.A0219114.N5aUCG.006.0656338553321.h5.mp",
+                  "bucket":"cumulus-test-sandbox-private",
+                  "fileName":"MOD09GQ.A0219114.N5aUCG.006.0656338553321.h5.mp",
+                  "path":"jk2-IngestGranuleSuccess-1558420117156-test-data/files",
+                  "url_path":"{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{substring(file.name, 0, 3)}",
+                  "type":"metadata",
+                  "duplicate_found":"True",
+                  "size":21708
+               }
+            ]
+         }
+      ],
+      "config":{
+         "glacier-bucket":"sndbx-cumulus-glacier",
+         "protected-bucket":"sndbx-cumulus-protected",
+         "internal-bucket":"sndbx-cumulus-internal",
+         "private-bucket":"sndbx-cumulus-private",
+         "public-bucket":"sndbx-cumulus-public",
+         "file-buckets":[
+            {
+               "regex":".*.h5$",
+               "sampleFileName":"L0A_HR_RAW_product_0010-of-0420.h5",
+               "bucket":"protected"
+            },
+            {
+               "regex":".*.cmr.xml$",
+               "sampleFileName":"L0A_HR_RAW_product_0010-of-0420.iso.xml",
+               "bucket":"protected"
+            },
+            {
+               "regex":".*.h5.mp$",
+               "sampleFileName":"L0A_HR_RAW_product_0001-of-0019.h5.mp",
+               "bucket":"public"
+            },
+            {
+               "regex":".*.cmr.json$",
+               "sampleFileName":"L0A_HR_RAW_product_0001-of-0019.cmr.json",
+               "bucket":"public"
+            }
+         ],
+         "excludeFileTypes":[
+            ".cmr"
+         ]
+      }
+}
+```
+## Output
+The output of lambda handler returns a dictionary having a list of granules that consist of granuleID and keys.
```

Comment:
I would add a link and point to your output schema here in addition to your example.

Reply:
added